pkg/controller/defaults/placements.go

```diff
@@ -125,20 +127,34 @@ var (
 				},
 			},
 			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
```

Comment:
In this case, if there's lesser nodes available, pod will never come up. Is that the desired behaviour?

Reply:
Isn't that the same issue for CephFilesystem already?

Reply:
I think this makes sense, yes.